plugins.json

```diff
@@ -527,10 +527,18 @@
         "name": "Tasmoticz",
         "repository": "https://github.com/joba-1/Tasmoticz"
     },
+    "Tuya": {
+        "author": "guino",
+        "branch": "master",
+        "description": "Controls TUYA devices your network (mainly on/off switches and Lights).",
+        "folder": "tuyaha",
+        "name": "Tuya",
+        "repository": "https://github.com/guino/tuyaha"
+    },
     "WuDirect": {
         "author": "vaneeten",
         "branch": "master",
-        "description": "Direct reception and parsing of Wunderground-formatted Weatherstation data.",
+        "description": "Controls TUYA devices your network (mainly on/off switches and Lights).",
```

Comment:
@galadril Looks like by accident you've updated the other plugin as well

Reply:
Woops good spot.. should be fixed now